vmu.py

```diff
@@ -33,48 +33,32 @@ def load_run_params(param_dir):
             # skip non-yaml files
             pass
         else:
-            if param_contents.has_key('platform') \
-               and param_contents.has_key('sources') \
-               and param_contents.has_key('packages'):
+            if set(param_contents.keys()) == set(['platforms', 'sources', 'package_sets']):
```

Comment:
Huh, that's neat. Though the one benefit of being explicit is clarity, particularly if you don't know that fun `set` and `dict` interaction.